### PR TITLE
Submission Feedback

### PR DESCRIPTION
**Pull request by aabrook**
This gets NPS Collect to a point where it can be used within supporter/other apps.

Jeffrey events are good to go
Design updates to match the latest mockups. Now with hearts!
The submit feedback button disables on submit.
The button and prompt disappear after a jeffrey submit is complete

screen shot 2016-11-03 at 3 18 50 pm
screen shot 2016-11-03 at 3 19 09 pm
screen shot 2016-11-03 at 3 19 26 pm

### CODE REVIEW

File: source/components/FeedbackIcon/styles.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { colors } from '../../traits'
```
Comment (kizza) [inline source/components/FeedbackIcon/styles.js]: Looks like colors is not required?

File: source/components/NPSCollect/index.js [hunk 2]
```diff
@@ -66,7 +66,7 @@ class NPSCollect extends React.Component {
           <img {...images.logo} className={css(styles.headerImg)} />
         </header>
 
-        <Preamble scoreSelected={scoreSelected} />
+        <Preamble score={score} />
```
Comment (kizza) [inline source/components/NPSCollect/index.js]: :+1:

File: source/components/FeedbackIcon/styles.js [hunk 3]
```diff
@@ -1,5 +1,3 @@
-import { colors } from '../../traits'
```
Comment (kizza) [inline source/components/FeedbackIcon/styles.js]: Argh, I fell for it again :)  It's removed now :+1:

File: source/components/NPSCollect/index.js [hunk 4]
```diff
@@ -15,23 +15,34 @@ class NPSCollect extends React.Component {
...
+    }
 
     this.submitFeedback = this.submitFeedback.bind(this)
     this.submitScore = this.submitScore.bind(this)
+    this.submitError = this.submitError.bind(this)
     this.handleFeedbackChanged = this.handleFeedbackChanged.bind(this)
     this.handleScoreSelected = this.handleScoreSelected.bind(this)
   }
 
+  submitError (err) {
```
Comment (kizza) [inline source/components/NPSCollect/index.js]: `submissionError`?  I don't feel super strongly about it but `.catch(this.submitError)` made me think it was posting to sentry or something :shrug:

File: source/components/NPSCollect/index.js [hunk 5]
```diff
@@ -42,7 +41,8 @@ class NPSCollect extends React.Component {
     this.setState({ sending: true })
 
     return sendNPSFeedback(pageId, userId, feedback).then(() => {
-      this.setState({ submitted: true, sending: false })
+      console.log('Sent')
```
Comment (kizza) [inline source/components/NPSCollect/index.js]: Are you wanting to keep this?

Comment (aabrook) [inline source/components/NPSCollect/index.js]: Already gone! I need to do more fixups/patches

File: source/data/__tests__/data-test.js [hunk 6]
```diff
@@ -44,6 +44,21 @@ describe('data sending module', () => {
     ))
   })
 
+  it('should call jeffrey.trackAction with a properly formatted submit feedback event', () => {
+    sendNPSFeedback({ pageId: 'test-page', userId: 'test-user', score: 3, feedback: 'Love you guys!' })
+
+    assert(jeffrey.trackAction.called)
+    assert(jeffrey.trackAction.calledWith(
```
Comment (kizza) [inline source/data/__tests__/data-test.js]: Is `called` followed by `calledWith` not redundant? (as in if it's "called with", it must have been called?)

**Comment by kizza** [comment]
Great stuff @aabrook.  Couple of optional comments.  I'd at minimum remove commits titled "stash" etc, other rebasing wouldn't need to hold this up if you'd prefer not to :+1:

**Comment by aabrook** [comment]
Squash and rebase all done @kizza

**Pull request merged**